queryviews/models.py

```diff
@@ -110,6 +110,8 @@ def get_random_sample(n, model, filters):
     # Our goal is to never actually execute this query
     query = get_query(model, filters)
     count = get_count(model, filters)
+    if count == 0:
```

Comment:
Needs to handle not just empty results, but count < n.  (which would result in the 2nd arg to randint being negative)

Reply:
There can't be a negative number of rows in a table...

Reply:
No but there can be less than 20.

Reply:
i realize i was reading this wrong in the first place, and it should be fixed now!